src/Pelagos/Bundle/AppBundle/Resources/public/js/search.js

```diff
@@ -34,4 +53,13 @@ $(document).ready(function () {
             href: url + "{{number}}"
         });
     }
+
+    // load qTip descriptions
+    $(".groupName").hover().each(function() {
+        $(this).qtip({
+            content: {
+                text: $.trim($(this).next().text())
```

Comment:
It's better to but something in .next(), like a selector, because elements might move around.